LondonBritishLibrary/orient/BLorient501.xml

```diff
@@ -175,11 +175,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="133r" to="140v"/>
+                                </item>
+                                <item xml:id="q19" n="19">
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="141r" to="148v"/>
+                                </item>
+                                <item xml:id="q20" n="20">
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="149r" to="156v"/>
+                                </item>
+                                <item xml:id="q21" n="21">
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="157r" to="164v"/>
+                                </item>
+                                <item xml:id="q22" n="22">
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="165r" to="172v"/>
+                                </item>
+                                <item xml:id="q23" n="23">
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="173r" to="180v"/>
+                                </item>
+                                <item xml:id="q24" n="24">
+                                    <dim unit="leaf">8</dim>
+                                    <locus from="181r" to="188v"/>
+                                </item>
+                                <item xml:id="q25" n="25">
+                                    <dim unit="leaf">7</dim>
+                                    <locus from="189r" to="195v"/>
+                                    3, stub after 6
+                                    4, stub after 6
+                                    <note>The quire structure is not clear from the images.</note>
+                                </item>
+                                    </list>
+                                </collation>
+                                <condition key="good">Stained by water and otherwise slightly damaged.</condition>
                             </supportDesc>
                             <layoutDesc>
-                                <layout columns="2" writtenLines="28 40"/>
+                                <layout columns="2" writtenLines="28 40">
+                                    <ab type="pricking">Pricking and ruling are visible.</ab>
+                                    <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
```

Comment:
I guess crux ansata is usually part of layout (?)
`<ab type="CruxAnsata"></ab>
<ab type="coronis"></ab>`

Reply:
Is it a part of layout?